src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Alerts and Applied Intelligence notification integrations
+tags:
+  - Alerts and Applied Intelligence
+  - Applied intelligence
+  - Incident Intelligence
+  - Destinations
+metaDescription: "Alerts and Applied Intelligence notification integrations are specific services and platforms you can use to send notifications from New Relic."
+---
+
+<Callout title="Early access">
+The features described here are early access.
+
+For more information on related features, see our docs on [Alerts notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/), [Incident Intelligence destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/), and [Proactive Detection notifications](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence/#set-up).
+</Callout>
+
+Alerts and Applied Intelligence notification integrations are specific services and platforms you can use to send notifications from New Relic.
+
+## Notification integration detail [#detail]
+
+Read about more about each of our specific notification integrations.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="email"
+    title="Email"
+  >
+  Send email notifications to users with or without New Relic accounts. You don't need to create new destinations for emails, as they're automatically added from email addresses in the event template.
+
+## Configure the event template [#configure-event-template]
+
+Users with New Relic accounts can be found via autocomplete when searching for first or last name.
+
+To add a user without a New Relic account or email distribution list, add the full email address.
+
+## Send a test notification [#email-test]
+
+You can send a test notification with a pre-defined example title and body to the emails you define.
+  </Collapser>
+    <Collapser
+    className="freq-link"
+    id="jira"
+    title="Jira"
+  >
+
+Send notifications to users of Atlassian Jira.
+
+## Permissions [#permissions]
+
+The required permissions from the Jira `API-Token` are `create`, `edit`, and `close` tickets.
+
+To enable the two-way sync toggle, the provided Jira `API-Key` needs  an `Admin` role.
+
+## Sett up a Jira destination [#set-up-jira]
+
+Create Jira issues, then enable Jira and New Relic to share updates and stay synced.
+
+To create a Jira destination, enter the following information:
+
+* :**Destination Name::** Custom name to identify the destination
```

Comment:
Are the : part of the style for each of these? They show in the build. if they are part of the style, then use code format to indicate this or provide an overall example to show this.

Also, I'd go with "Destination name" and "API token" unless those are coded into the process. I assume they are coded in to reflect Jira.

Reply:
Thanks! On thinking about it, I don't think we need these extra colons. I fixed the capitalization too.